tsfc/fem.py

```diff
@@ -522,11 +543,12 @@ def callback(key):
 
 
 def coordinate_coefficient(domain):
+    """Create a fake coordinate coefficient for a domain."""
     return ufl.Coefficient(ufl.FunctionSpace(domain, domain.ufl_coordinate_element()))
 
 
 def replace_coordinates(integrand, coordinate_coefficient):
-    # Replace SpatialCoordinate nodes with Coefficients
+    """Replace SpatialCoordinate nodes with Coefficients."""
     return map_expr_dag(ReplaceSpatialCoordinates(coordinate_coefficient), integrand)
```

Comment:
I wonder if we should map `replace_coordinates` over all the integrands in an integral (rather than just singly) ?